src/buildApplication/buildWorkerFile.ts

```diff
@@ -83,7 +83,7 @@ const nodeBufferPlugin: Plugin = {
 	name: 'node:buffer',
 	setup(build) {
 		build.onResolve({ filter: /^node:buffer$/ }, ({ kind, path }) => {
-			if (kind === 'require-call') return;
+			if (kind !== 'require-call') return;
```

Comment:
How about:

```ts
if (kind === 'require-call') {
  return { path, namspace: 'node-buffer' };
}
```

Reply:
With a comment explaining what this is here for.

Reply:
I think I did try that but seems like typescript complains:
![Screenshot 2023-04-25 at 10 19 10](https://user-images.githubusercontent.com/61631103/234232711-d64db112-b42f-46c6-aec3-a34c9f8c8c78.png)

I can for example add an extra return to make typescript happy...
![Screenshot 2023-04-25 at 10 20 31](https://user-images.githubusercontent.com/61631103/234232929-9a95d29a-32db-4bcd-8767-854f974cf1cf.png)

but that looks kind of awkward, doesn't it? 😓

Reply:
Well that is a bit stupid of TypeScript. How about:

```ts
return kind === 'require-call' ? { path, namespace: 'node-buffer' } : undefined;
```

Reply:
ok I've applied that, updated my comment and added a new comment to the `onLoad`

so hopefully things should be much clearer now, please have another look 🙏 🙂